### DISCUSSION TITLE
Add a feature to also report the spammer's account

### DISCUSSION THREAD

**Discussion by bloopy2** [ThioJoe/YT-Spammer-Purge#276] [Ideas]
I think there could be a feature that not only reports/deletes a spammer's comment, but also reports the spammer's account.

**Comment by kiatchun** [comment]
> I think there could be a feature that not only reports/deletes a spammer's comment, but also reports the spammer's account.

This feature has already been requested multiple times, but has never been implemented.

**Reply by bloopy2** [reply]
oh ok
